### PR TITLE
Clean the OpenShift template from Tenant Manager

### PR DESCRIPTION
**Pull request by andreaTP**
Ref:
https://github.com/Apicurio/apicurio-tenant-manager/pull/64

### CODE REVIEW

File: distro/openshift-template/mt/apicurio-registry-mt-template.yaml [hunk 1]
```diff
@@ -780,23 +472,13 @@ parameters:
 
 - name: TENANT_MANAGER_URL
   value: http://tenant-manager:8585
-- name: TENANT_MANAGER_CA_BUNDLE
```
Comment (carlesarnal) [inline distro/openshift-template/mt/apicurio-registry-mt-template.yaml:783]: Actually, the CA bundle must be preserved.

File: distro/openshift-template/mt/apicurio-registry-mt-template.yaml [hunk 2]
```diff
@@ -780,23 +472,13 @@ parameters:
...
-  description: path to tenant-manager service CA bundle
-- name: TENANT_MANAGER_CLIENT_SECRET_NAME
```
Comment (carlesarnal) [inline distro/openshift-template/mt/apicurio-registry-mt-template.yaml:785]: Same with the client secret name, Registry needs these credentials to access TM.

**Review by carlesarnal — CHANGES_REQUESTED** [review]
Certificate and auth-related env vars must be kept in Registry.

File: distro/openshift-template/mt/apicurio-registry-mt-template.yaml (hunk 1 above)
Comment (andreaTP) [reply distro/openshift-template/mt/apicurio-registry-mt-template.yaml]: restored

File: distro/openshift-template/mt/apicurio-registry-mt-template.yaml (hunk 2 above)
Comment (andreaTP) [reply distro/openshift-template/mt/apicurio-registry-mt-template.yaml:785]: restored

**Review by carlesarnal — APPROVED** [review]
Good now!

**Pull request merged**